spec/integration/manifest.spec.yml

```
@@ -225,13 +225,21 @@ spec:
             enum:
               - action
               - next_step
+          appId:
```

We use snake_case for attributes here.
```suggestion
          app_id:
```